eng/scripts/typespec/Generate-Code.ps1

```diff
@@ -4,21 +4,24 @@
 [CmdletBinding()]
 param(
   [Parameter(Mandatory=$false)]
-  [string] $ClangFormatPath = "clang-format"
+  [string] $ClangFormatPath = "clang-format",
+
+  [Parameter(Mandatory=$false)]
+  [string] $outputPath = ".\generated"
 )
 
 $ErrorActionPreference = "Stop"
 $PSNativeCommandUseErrorActionPreference = $true
 
 pushd
 cd $PSScriptRoot
-cd ../
+cd ../../../
```

Comment:
If you're trying to get to the root of the repo I recommend something like `Set-Location $PSScriptRoot../../../` (make sure you use the correct number of `../`s). 

This location will work even if you change the directory context in which the script is running.

Reply:
the scripts are first copied in the typespec folder after it has been brought down via git , they are run to invoke codegen to specifically build the typespec packages and run them afterwards. Once we publish those to NPM these scrip[ts won't be needed anymore.